### ISSUE TITLE
sqlite3.OperationalError

### ISSUE THREAD

**Issue by Purzelkater** [Kruptein/PlanarAlly#100]
I have made a fresh clean installation on my system with Windwos 10. Python 3.7.1 was fresh installed and dependencies installed with _pip install -r requirements.txt_.

After _python planarserver.py_ I get this error:
```
Traceback (most recent call last):
  File "D:\Python\Python37\lib\site-packages\peewee.py", line 2683, in execute_sql
    cursor.execute(sql, params or ())
sqlite3.OperationalError: no such table: constants
```

**Comment by Kruptein** [comment]
I apparently never properly tested it when someone runs the server without converting a previous save.  I've fixed it already but am checking if I notice some other bugs before pushing my changes

**Comment by Kruptein** [comment]
Alright it's fixed and published as 0.11.1
